Applications/Games/Elite:Dangerous/Steam/script.js

```diff
@@ -0,0 +1,23 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+include(["Engines", "Wine", "Verbs", "dotnet452"]);
+include(["Engines", "Wine", "Verbs", "vcrun2012"]);
+include(["Engines", "Wine", "Verbs", "quartz"]);
+
+new SteamScript()
+    .name("Elite:Dangerous")             
+    .editor("Frontier Developments")     
+    .author("ImperatorS79")
+    .wineVersion(LATEST_STAGING_VERSION)
+    .wineDistribution("staging")
+    .wineArchitecture("amd64")
+    .preInstall(function(wine, wizard) {
+        wine.dotnet452();
+        wine.corefonts();
+        wine.quartz();
+        wine.vcrun2012();
+        wine.windowsVersion("win7");
```

Comment:
Isn't this the default anyways?

Reply:
As I wanted it to be as close as possible to winetricks source, the dotnet452 verb set the windows version to 2003.

Reply:
Ok. Didn't see that in the review. I don't think we should change the windows version from within a verb (because you cannot tell from the outside).

Reply:
@ImperatorS79 can you provide a PR which removes that dotnet452 changes the windows version internally (you can show a warning if it's not 2003) and then remove `wine.windowsVersion("win7");` in this PR? I would merge it then.